tests/test_nap.py

```
@@ -0,0 +1,408 @@
+    before = _payload_names(repo)
+    with pytest.raises(ValueError, match="overlapping packs") as caught:
+        m.write_nap(repo, ids[0], ids[1], "nope")
+    _agent_err(str(caught.value))
+    assert _payload_names(repo) == before
+
+
+def test_write_nap_note_inside_adjacent_nap_raises(tmp_path):
+    """A note whose digest sits in the adjacent nap is overlapping packs."""
+    m = load_summem()
+    repo = init_repo(tmp_path / "r")
+    pa, pb = _two_notes(m, repo)
```

Ruff (0.16.1)

[warning] Unpacked variable `pb` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)